### PR TITLE
Simple javax.servlet.Servlet implementation

### PR DESCRIPTION
**Pull request by zsigmond-czine-everit**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Import-Package>
+              *
```
Comment (balazs-zsoldos) [inline pom.xml:85]: import javax.servlet.\* with range "[2.4,4)"

Comment (zsigmond-czine-everit) [inline pom.xml:85]: done

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Import-Package>
+              javax.servlet.*;version="[2.4,4)"
+            </Import-Package>
```
Comment (balazs-zsoldos) [inline pom.xml:86]: and add \* of course to import every other (non javax.servlet ones) necessary packages ;-)

Comment (zsigmond-czine-everit) [inline pom.xml:86]: Added.

**Pull request merged**